core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-admin-page.import.ts

```diff
@@ -19,6 +19,11 @@
 import 'core-js/es7/reflect';
 import 'zone.js';
 
+// This throws "TS2307". We need to
```

Comment:
Can you explain why this behaves differently from e.g. core-js, zone.js, etc. above?

Reply:
![Screenshot from 2023-01-10 20-57-24](https://user-images.githubusercontent.com/80846488/211592550-c2bf622d-8421-415d-b59a-f2ec54004604.png)

most probably npm don't contain types of uivalidate, not sure about that. So add comment that, we can't import it directly like `angular-ui-validate`.

Reply:
Can you add stubs for this library in _typings_ folder?

Reply:
Yeah -- @JeeveshGarg I think the error message tells you how to fix this?

Reply:
Done.